### PR TITLE
Wrap words for better readability

### PR DESCRIPTION
**Pull request by AvdLee**
Long text would not easily be readable since it requires us to scroll horizontally. This is a quick win that will improve readability

image

### CODE REVIEW

File: Sources/Logging/Loggable.swift [hunk 1]
```diff
@@ -2,6 +2,13 @@ import Foundation
 import MetricKit
 
 protocol Loggable {
+
```
Comment (wetransferplatform) [inline Sources/Logging/Loggable.swift:5]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

File: Sources/Logging/Loggable.swift [hunk 2]
```diff
@@ -10,15 +17,30 @@ protocol Loggable {
...
 
     var logData: Data? {
         if let cssClass = cssClass {
-            return "<p class=\"\(cssClass)\">\(message)</p>".data(using: .utf8)
+            return "<p class=\"\(cssClass)\">\(logMessage)</p>".data(using: .utf8)
         } else {
             return message.data(using: .utf8)
         }
     }
+
```
Comment (wetransferplatform) [inline Sources/Logging/Loggable.swift:31]: - :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Testing cancelled because the build failed.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Some model objects have unexpectedly changed since the build was started. Please try again.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Build service could not create build operation: internal error: could not generate PIF because the workspace has not finished loading or is still waiting for package resolution</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **Sources/Logging/Loggable.swift#L8** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/Logging/Loggable.swift#L11** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/405d7373-bfa9-4334-aedc-d4bac78b7649" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against f3ccc5cc449b9dbb937f7c2dd1b958d89cd11434
</p>

**Review by kairadiagne — APPROVED** [review]

**Review by kevinrenskers — APPROVED** [review]
There are some SwiftLint warnings, but yeah code looks good to me

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 3.0.0](https://github.com/WeTransfer/Diagnostics/releases/tag/3.0.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**